common/djangoapps/mailchimp_pipeline/signals/handlers.py

```diff
@@ -88,23 +91,53 @@ def send_user_info_to_mailchimp(sender, user, created, kwargs):
 
 
 @task()
-def send_user_enrollments_to_mailchimp(sender, instance, created, kwargs):
+def task_send_user_info_to_mailchimp(data):
+    """ Create user account at nodeBB when user created at edx Platform """
+
+    user = User.objects.get(id=data['user_id'])
+    created = data["created"]
+
```

Comment:
please call the related function in this method instead of replicating

Reply:
Done